### DISCUSSION TITLE
0.173 in autostart?

### DISCUSSION THREAD

**Discussion by RubinDublone593** [seerge/g-helper#2697] [Q&A] [answered]
Theres the new verion 0.173, but i currently also have 0.170 installed, how do I get 0.173 to start automatically instead of 0.170?

**Comment by seerge** [comment, marked as answer]
@RubinDublone593 hello. Just replace old version exe with new one :) 

And if you have forgotten where you have placed old exe, just run a new one and uncheck / check ``Run on startup``. It would recreate startup task pointing to new exe location
